titan-compiler/coder.lua

```diff
@@ -1305,6 +1311,130 @@ $INCLUDES
 
 ]]
 
+local libopen = [[
```

Comment:
We could store this the preamble in a separate file.

Reply:
The problem is where to load it from when running the compiler. Remember that `coder.lua` is a regular Lua module, loaded by `require`. We would probably need another configurable path for the compiler, which is too much of a hassle until our "runtime" grows more.

Reply:
Yes, this don't need to be done now, I'll create a issue deal with the runtime.